index.html

```diff
@@ -8,7 +8,13 @@
 
     <link rel="apple-touch-icon" href="images/apple-touch-icon.png">
     <link rel="stylesheet" href="css/all.css">
-
+    <script>
+        document.addEventListener('DOMContentLoaded', () => {
```

Comment:
Won't this also trigger even when we failed to load some of the critical files?

Reply:
I will add a guard for the truthiness of JitsiMeetJS, but wouldn't we have bigger problems if that happens than some exception being thrown in the console?

Reply:
We have that inline HTML page telling you things are wrong, in principle things would stop there.